### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [devHudi/hudi.blog#209] [Comments]
# about/

꾸준히, 의미있는 학습을 기록하기 위한 공간입니다.

https://hudi.blog/about/

**Comment by devmeeple** [comment]
덕분에 알찬 블로그 생활 중입니다. 어떤 분 일지 궁금했는데 기능으로 추가해 주셔서 감사합니다. 
이번 업데이트도 잘 쓰겠습니다.

**Reply by devHudi** [reply]
안녕하세요 @devmeeple 님! gatsby-starter-hoodie 를 사용해주셔서 감사해요 ㅎㅎ About 페이지가 릴리즈되자마자 적용해주셨군요 👍 많은 피드백 부탁드립니다!

**Reply by Whatisthis8047** [reply]
덕분에 멋진 템플릿으로 기술 블로그를 시작해볼 수 있었습니다! ㅎㅎ 감사합니다!!

**Reply by devHudi** [reply]
@Whatisthis8047 님 따뜻한 댓글 감사합니다 🙂
